LazyProxy.Autofac/AutofacExtensions.cs

```diff
@@ -50,7 +29,7 @@ static AutofacExtensions()
             builder.RegisterLazy(typeof(TFrom), typeof(TTo), name, nonLazyRegistrationMutator);
 
         /// <summary>
-        /// Is used to register interface TFrom to class TTo by creation a lazy proxy at runtime.
+        /// Is used to register non open generic interface TFrom to class TTo by creation a lazy proxy at runtime.
```

Comment:
It would be good to replace the "Is used to register" with just "Registers", and also the text should answer the question "Registers what?".
In this concern, the text would be "Registers a mapping of a non-open generic interface TFrom to the class TTo via run-time generated lazy proxy type."

Reply:
Yes, that's definitely better, I will change.